scripts/get_model_image.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+__author__ = "Jurjen de Jong"
+
+import os
+import re
+from argparse import ArgumentParser
+from glob import glob
+from itertools import repeat
+
+from casacore.tables import table
+from astropy.io import fits
+
+
+def add_trailing_zeros(s: str, digitsize: int = 4) -> str:
+    """
+    Pad a string with leading zeros to a fixed width.
+
+    Parameters
+    ----------
+    s : str
```

Comment:
```suggestion
    s
```
The type is declared in the function signature, so in principle isn't required here